addons/project/models/project.py

```diff
@@ -1645,12 +1645,13 @@ def default_get(self, default_fields):
                 )
                 if partner_id:
                     vals['partner_id'] = partner_id
-        if vals.get('project_id'):
-            project = self.env['project.project'].browse(vals.get('project_id'))
+        project_id = vals.get('project_id', self.env.context.get('default_project_id'))
+        if project_id:
+            project = self.env['project.project'].browse(project_id)
             if project.analytic_account_id:
                 vals['analytic_account_id'] = project.analytic_account_id.id
             if project.analytic_tag_ids:
-                vals['analytic_tag_ids'] = [Command.set(project.analytic_tag_ids.ids)]
+                vals['analytic_tag_ids'] = [Command.link(tag.id) for tag in project.analytic_tag_ids]
```

Comment:
Is this really a required change?

Reply:
> Is this really a required change?

Yes is required because default analytics tags are not set. [see](https://www.awesomescreenshot.com/video/10316603?key=d25f2602e5687f6aca1820f6d73ad34e)

Reply:
> Is this really a required change?

Done. thanks